{
 "pytest_faulthandler": {
  "lineno": null,
  "symbols_in_volume": [],
  "type": "module"
 },
 "pytest_faulthandler._cancel_traceback_dump": {
  "lineno": 67,
  "symbols_in_volume": [
   "faulthandler.cancel_dump_traceback_later"
  ],
  "type": "function"
 },
 "pytest_faulthandler.pytest_addoption": {
  "lineno": 6,
  "symbols_in_volume": [],
  "type": "function"
 },
 "pytest_faulthandler.pytest_configure": {
  "lineno": 20,
  "symbols_in_volume": [
   "faulthandler.enable",
   "os.dup",
   "os.fdopen",
   "sys.__stderr__.fileno",
   "sys.stderr",
   "sys.stderr.fileno"
  ],
  "type": "function"
 },
 "pytest_faulthandler.pytest_enter_pdb": {
  "lineno": 75,
  "symbols_in_volume": [
   "pytest.hookimpl"
  ],
  "type": "function"
 },
 "pytest_faulthandler.pytest_exception_interact": {
  "lineno": 82,
  "symbols_in_volume": [
   "pytest.hookimpl"
  ],
  "type": "function"
 },
 "pytest_faulthandler.pytest_runtest_protocol": {
  "lineno": 50,
  "symbols_in_volume": [
   "faulthandler.cancel_dump_traceback_later",
   "faulthandler.dump_traceback_later",
   "pytest.mark.hookwrapper"
  ],
  "type": "function"
 },
 "pytest_faulthandler.timeout_support_available": {
  "lineno": 41,
  "symbols_in_volume": [
   "faulthandler"
  ],
  "type": "function"
 }
}